src/Controls/src/Core/TableSection.cs

```diff
@@ -155,6 +155,8 @@ void OnChildrenChanged(object sender, NotifyCollectionChangedEventArgs notifyCol
 		}
 
 		IReadOnlyList<Maui.IVisualTreeElement> IVisualTreeElement.GetVisualChildren() => this._children.Cast<IVisualTreeElement>().ToList().AsReadOnly();
+
+		IVisualTreeElement IVisualTreeElement.GetVisualParent() => null;
```

Comment:
We don't have direct access to the parent `TableRoot` or other `TableSection`s here. The other issue for Parent is that the underlying Cells all have Parents of the upper most 'TableView', so the _really_ implement it right, we would need to nest the calls all the way down for the right parents... but Cell is an `Element` and that `Parent` has to be `Element`, which the TableSection's are not. 

For Hot Reload and LVT, we don't need to call GetVisualParent for the section, so leaving it null is fine.